docs/plugins.md

```diff
@@ -103,81 +124,38 @@ module.exports = (snowpackConfig, pluginOptions) => ({
 
 This is a simplified version of the official Snowpack Svelte plugin, simplified to make the intent clearer.
 
-You don’t need to be familiar with Svelte, but in this example just know we want to take in Svelte files (`.svelte`) and generate JS & CSS from them. We can see how that’s reflected in the **input** (`['.svelte']`) and **output** (`['.js', '.css']`). When we run `svelte.compile()` we get back our CSS and JS, and pass that back as an object to Snowpack: `{ '.js': …, '.css': … }`.
+You don’t need to be familiar with Svelte, but in this example just know we want to take in Svelte files (`.svelte`) and generate JS & CSS for our final build. We can see how that’s reflected in the **resolve** input (`['.svelte']`) and output (`['.js', '.css']`). When we run `svelte.compile()` we get back our CSS and JS, and return an object with entires that match those **resolve** output entires: `{ '.js': …, '.css': … }`.
 
-Say the original input file was `src/components/App.svelte`. Because `.svelte` is in `input`, that file will trigger this plugin. `build()` will then execute, and Snowpack will take the `.js` and `.css` from the return object to generate 2 files: `src/components/App.js` and `src/components/App.css`. Snowpack kept the `App` filename and only changed the extension. Because your plugin only has to worry about individual files, and not the system at large, plugins are easier to write and maintain and easier to chain together.
+To see this in action, lets say we have a source file at `src/components/App.svelte`. Because the `.svelte` file extension is in our plugin `resolve.input` array, Snowpack knows that this plugin is responsible for loading that file in our build. `load()` will then execute, and Snowpack will take the `.js` and `.css` results from the return object to generate the 2 build results: `src/components/App.js` and `src/components/App.css`. Snowpack will always keep the original file name (`App`) and only ever change the extension in the build. 
```

Comment:
```suggestion
To see this in action, let's say we have a source file at `src/components/App.svelte`. Because the `.svelte` file extension is in our plugin `resolve.input` array, Snowpack knows that this plugin is responsible for loading that file in our build. `load()` will then execute, and Snowpack will take the `.js` and `.css` results from the return object to generate the 2 build results: `src/components/App.js` and `src/components/App.css`. Snowpack will always keep the original file name (`App`) and only ever change the extension in the build. 
```